synapse/lib/ast.py

```
@@ -424,7 +439,12 @@ def getRuntVars(self, runt):
 
         async for node, path in genr:
 
-            async for item in s_coro.agen(await self.kids[1].compute(path)):
+            # TODO: remove when storm is all objects
+            valu = await self.kids[1].compute(path)
+            if isinstance(valu, dict):
+                valu = valu.items()
```

this is dangerous since you can raise an exception inside of iteration